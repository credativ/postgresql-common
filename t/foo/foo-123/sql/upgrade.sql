CREATE EXTENSION foo VERSION "100";

SELECT foo();

ALTER EXTENSION foo UPDATE TO "123";

SELECT foo();
