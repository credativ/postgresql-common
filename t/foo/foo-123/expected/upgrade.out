CREATE EXTENSION foo VERSION "100";
SELECT foo();
   foo   
---------
 old bar
(1 row)

ALTER EXTENSION foo UPDATE TO "123";
SELECT foo();
 foo 
-----
 bar
(1 row)

